deluge/tests/test_web_api.py

```diff
@@ -76,7 +76,7 @@ def test_set_config(self):
         assert config['pwd_sha1'] != web_config['pwd_sha1']
         assert config['sessions'] != web_config['sessions']
 
-    @defer.inlineCallbacks
+    @pytest_twisted.inlineCallbacks
```

Comment:
These changes are due to the pytest developers recommending that twisted not be used directly in tests with fixtures. From the pytest-twisted docs:

`Using twisted.internet.defer.inlineCallbacks as a decorator for test functions, which use fixtures, does not work. Please use pytest_twisted.inlineCallbacks instead.`